bundles/org.openhab.binding.sonyprojector/src/main/java/org/openhab/binding/sonyprojector/internal/communication/SonyProjectorCalibrationPreset.java

```diff
@@ -144,6 +157,27 @@ public static List<StateOption> getStateOptions(int category) {
         return options;
     }
 
+    /**
+     * Get the list of {@link CommandOption} associated to the available calibration presets for a particular category
+     * of
+     * projector models
+     *
+     * @param category a category of projector models
+     *
+     * @return the list of {@link CommandOption} associated to the available calibration presets for a provided category
+     *         of projector models
+     */
+    public static List<CommandOption> getCommandOptions(int category) {
+        List<CommandOption> options = new ArrayList<>();
+        for (SonyProjectorCalibrationPreset value : SonyProjectorCalibrationPreset.values()) {
+            if (value.getCategory() == category) {
+                options.add(new CommandOption("PRESET_" + value.getName().toUpperCase(),
+                        "@text/channel-type.sonyprojector.calibrationpreset.state.option." + value.getName()));
```

Comment:
Is it possible for such key to not exist so that the key would be displayed rather than a human-readable text?

Reply:
No, it is not possible because I checked there is already an entry for each enum value  I am reusing the existing keys already defined for this enum for the state options of another channel